apisix/plugins/file-logger.lua

```diff
@@ -38,6 +38,13 @@ local schema = {
             items = {
                 type = "array"
             }
+        },
+        matches = {
```

Comment:
I think `match` is better than `matches`

Reply:
@monkeyDluffy6017 wouldn't it be better to use the plural form here?

Reply:
I see the `match` is used everywhere in the APISIX, so just keep consistent

Reply:
okok